main.go

```diff
@@ -95,9 +95,10 @@ type controllerWebhookConfig struct {
 }
 
 type safeModeConfig struct {
-	Enable             bool `json:"enable"`
-	NamespaceThreshold int  `json:"namespaceThreshold"`
-	ClusterThreshold   int  `json:"clusterThreshold"`
+	SpecifiedEnvironment string `json:"specifiedEnvironment"`
```

Comment:
```suggestion
	Environment string `json:"specifiedEnvironment"`
```

Same comment as for the disruption spec, why couldn't we shorten that to `Environment`?